test/openid_connect_test.exs

```diff
@@ -446,6 +446,35 @@ defmodule OpenIDConnectTest do
       assert verify(config, token) == {:ok, claims}
     end
 
+    # This test is only needed due to the following issue in erlang-jose:
+    # https://github.com/potatosalad/erlang-jose/issues/177
+    #
+    # Prior to the code changes that prompted adding this test, if an EdDSA
+    # key was present in the JWKS prior to the key that signed the JWT
+    # the `verify_signature` function, would return an {:error, reason} and would
+    # cause the function to return early without finding the appropriate signing key.
+
+    test "returns claims when JWKS contains EdDSA key prior to signing key" do
+      {jwks, []} = Code.eval_file("test/fixtures/jwks/jwks_eddsa.exs")
+      {jwk, []} = Code.eval_file("test/fixtures/jwks/jwk.exs")
+
```

Comment:
(nit): I think it might be a little cleaner to have these as either module attributes, or returned from a `Fixtures` module function so that we don't need to resort to `Code.eval_file` at runtime.

Reply:
I don't disagree.  I was just following what was done for all of the other tests.  Are you asking that I update all of the tests with you suggestion?

Reply:
Ah, this is a convention. Ok, disregard. May not make sense to update these if the goal is to track upstream.